File: app-code/backend/src/main/java/com/sanfernando/sanfernando/dtos/requests/PedidoRequest.java
package com.sanfernando.sanfernando.dtos.requests;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class PedidoRequest {
  private Integer idRepresentante;
  private Integer idEmpleadoRegistro;
  private String idTipoPedido;
  private String idEstadoPedido;
  private Integer idTicket;
  private String fechaRegistro;
  private String fechaEntrega;
  private PedidoTicketProductoRequest[] pedidoTicketProductoRequest;
}

File: app-code/frontend/src/app/core/models/request/pedido-request.ts
import { TicketProductoRequest } from "./ticket-producto-request";

export interface PedidoRequest {
  idRepresentante?: number;
  idEmpleadoRegistro?: number;
  idTipoPedido?: string;
  idEstadoPedido?: string;
  fechaRegistro?: string;
  fechaEntrega?: string;
  pedidoTicketProductoRequest?: TicketProductoRequest[];
}
